apis/secscan/v1alpha1/types.go

```diff
@@ -45,7 +45,7 @@ type ImageManifestVulnStatus struct {
 	FixableCount    int `json:"fixableCount,omitempty"`
 
 	// Map from pod's path to container ids
-	AffectedPods map[string][]string `json:"afftectedPods,omitempty"`
+	AffectedPods map[string][]string `json:"affectedPods,omitempty"`
```

Comment:
Found this typo thanks to the effect of pruning unused fields when using structural schemas!